src/components/todoForm/index.jsx

```diff
@@ -0,0 +1,35 @@
+import TodoList from "../../../src/containers/todoListContainer";
+
+export const TodoForm = ({todos, addTodoHandler }) => {
+
+  const addTodoHandlerWrapper = (e) => {
+    e.preventDefault();
+    const name = e.target[0].value;
+    if (name.trim() !== "") {
+      addTodoHandler({name: name, completed: false});
+    }
+    e.target[0].value = "";
```

Comment:
what's the purpose for setting event as empty ?

Reply:
It's because when the todo is added, I'm clearing the input field using an event.